core/src/main/java/org/jdbi/v3/core/mapper/MapEntryRowMapper.java

```diff
@@ -36,7 +36,7 @@
  * @param <K> entry key type
  * @param <V> entry value type
  */
-public class MapEntryMapper<K, V> implements RowMapper<Map.Entry<K, V>> {
+public class MapEntryRowMapper<K, V> implements RowMapper<Map.Entry<K, V>> {
```

Comment:
Not sure I agree with this rename. With the exception of `JoinRowMapper` (where `Row` belongs to `Join` more than `Mapper` IMO), all the `RowMapper` implementations omit `Row` from their names.